kotlin/android/app/build.gradle.kts

```diff
@@ -172,16 +169,12 @@ dependencies {
     implementation("androidx.lifecycle:lifecycle-livedata-ktx:2.6.1")
 
     // Navigation
-    implementation("androidx.navigation:navigation-fragment-ktx:$navVersion")
-    implementation("androidx.navigation:navigation-ui-ktx:$navVersion")
-
-    // Safe Args
-    //
+    implementation("androidx.navigation:navigation-fragment-ktx:2.7.6")
+    implementation("androidx.navigation:navigation-ui-ktx:2.7.6")
 
     // Hilt
-    implementation("com.google.dagger:hilt-android:2.50")
+    implementation("com.google.dagger:hilt-android:2.51")
```

Comment:
And this change is just manually updating this one dep?

Reply:
Yeah, dependabot was trying to update them separately, so rather than waiting a month for it to try again I just went ahead and bumped it.